### PR TITLE
chore(doc): add jest specific pckg command

### PR DESCRIPTION
**Pull request by tony-go**
| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            | ❌
| Patch: Bug Fix?          | ❌
| Major: Breaking Change?  | ❌
| Minor: New Feature?      | ❌
| Tests Added + Pass?      | ❌
| Documentation PR Link    | ❌
| Any Dependency Changes?  | ❌
| License                  | MIT

Hi 👋 

Yesterday, chatting with @JLHwung we figured out that the command: `yarn jest <name-of-the-pckg> --watch` wasn't mentioned in the documentation.

A tiny PR to add it ^^

Let me know if it's useful or not.

<a href="https://gitpod.io/#https://github.com/babel/babel/pull/13607"></a>

### CODE REVIEW

**Review by fedeci — COMMENTED** [review]
Since there are equivalent flags also for `TEST_GREP` and `TEST_DEBUG`, why don't we document all of them?

**Comment by tony-go** [comment]
> Since there are equivalent flags also for `TEST_GREP` and `TEST_DEBUG`, why don't we document all of them?

Just to mention that the command `TEST_DEBUG=true make test` and `yarn node --inspect-brk node_modules/.bin/jest --runInBand` doesn't work on my machine 🙈

**Comment by nicolo-ribaudo** [comment]
TIL about `TEST_DEBUG=true` :grimacing:

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -140,6 +152,12 @@ To enable the Node.js debugger added in v6.3.0, set the `TEST_DEBUG` environment
 $ TEST_DEBUG=true make test
 ```
 
+Or you can use Node
```
Comment (fedeci) [inline CONTRIBUTING.md:155]:
```suggestion
Or you can use Node:
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -140,6 +152,12 @@ To enable the Node.js debugger added in v6.3.0, set the `TEST_DEBUG` environment
...
+
+```sh
+yarn node --inspect-brk node_modules/jest/bin/jest.js --runInBand
+```
```
Comment (fedeci) [inline CONTRIBUTING.md:159]:
```suggestion
Or you can use Yarn:

```sh
$ yarn node --inspect-brk node_modules/jest/bin/jest.js --runInBand
```
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -122,12 +122,24 @@ $ TEST_ONLY=babel-cli make test
...
 Use the `TEST_GREP` variable to run a subset of tests by name:
 
 ```sh
 $ TEST_GREP=transformation make test
 ```
 
+Or you can use Jest:
+
+```sh
+yarn jest -t transformation
```
Comment (fedeci) [inline CONTRIBUTING.md:140]:
```suggestion
$ yarn jest -t transformation
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -122,12 +122,24 @@ $ TEST_ONLY=babel-cli make test
 $ TEST_ONLY=babel-plugin-transform-classes make test
 ```
 
+Or you can use Jest:
+
+```sh
+yarn jest babel-cli
```
Comment (fedeci) [inline CONTRIBUTING.md:128]:
```suggestion
$ yarn jest babel-cli
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -122,12 +122,24 @@ $ TEST_ONLY=babel-cli make test
 $ TEST_ONLY=babel-plugin-transform-classes make test
 ```
 
+Or you can use Jest:
```
Comment (fedeci) [inline CONTRIBUTING.md:125]:
```suggestion
Or you can use Yarn:
```
nit: the runner is Yarn, not Jest :)

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -122,12 +122,24 @@ $ TEST_ONLY=babel-cli make test
...
+yarn jest babel-cli
+```
+
 Use the `TEST_GREP` variable to run a subset of tests by name:
 
 ```sh
 $ TEST_GREP=transformation make test
 ```
 
+Or you can use Jest:
```
Comment (fedeci) [inline CONTRIBUTING.md:137]:
```suggestion
Or you can use Yarn:
```

**Review by fedeci — COMMENTED** [review]
Apart from the nitpicks, I was thinking about a more tabular view of this commands, what's your point of view?

**Comment by tony-go** [comment]
Fix nitpicks and reshape a bit the documentation. 

[👀](https://github.com/tony-go/babel/blob/doc-test-only/CONTRIBUTING.md)

Let me know.

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -82,72 +82,110 @@ $ make build-dist
...
 When working on an issue, you will most likely want to focus on a particular [packages](https://github.com/babel/babel/tree/main/packages). Using `TEST_ONLY` will only run tests for that specific package.
 
 ```sh
 $ TEST_ONLY=babel-cli make test
 ```
 
-`TEST_ONLY` will also match substrings of the package name:
+<details>
+  <summary>More options</summary>
+  `TEST_ONLY` will also match substrings of the package name:
```
Comment (fedeci) [inline CONTRIBUTING.md:122]:
```suggestion
  <code>TEST_ONLY</code> will also match substrings of the package name:
```
markdown cannot still render code blocks properly when nested inside `<details>`.

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -82,72 +82,110 @@ $ make build-dist
...
-$ TEST_GREP="arrow functions destructuring parameters" make test
-```
+  ```sh
+  $ yarn jest -t transformation
+  ```
+
+  Substitute spaces for hyphens and forward slashes when targeting specific test names:
+
+  ```sh
+  $ TEST_GREP="arrow functions destructuring parameters" make test
```
Comment (fedeci) [inline CONTRIBUTING.md:156]: nit: we should keep the same order of the above, yarn after make

**Review by fedeci — COMMENTED** [review]
Great🚀

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -82,72 +82,110 @@ $ make build-dist
...
+<br>
+
 You can combine `TEST_DEBUG` with `TEST_GREP` or `TEST_ONLY` to debug a subset of tests. If you plan to stay long in the debugger (which you'll likely do!), you may increase the test timeout by editing [test/testSetupFile.js](https://github.com/babel/babel/blob/main/test/testSetupFile.js).
 
 To overwrite any test fixtures when fixing a bug or anything, add the env variable `OVERWRITE=true`
 
-```sh
-$ OVERWRITE=true TEST_ONLY=babel-plugin-transform-classes make test-only
+  ```sh
+  $ OVERWRITE=true TEST_ONLY=babel-plugin-transform-classes make test-only
```
Comment (fedeci) [inline CONTRIBUTING.md:184]:
```suggestion
```sh
$ OVERWRITE=true TEST_ONLY=babel-plugin-transform-classes make test-only
```
The last one I promise!

**Review by fedeci — APPROVED** [review]

File: CONTRIBUTING.md (hunk 9 above)
Comment (tony-go) [reply CONTRIBUTING.md]: Ahahah! I forgot it, my bad ^^

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -82,64 +82,100 @@ $ make build-dist
...
-To enable the Node.js debugger added in v6.3.0, set the `TEST_DEBUG` environment variable:
+To enable the Node.js debugger added in v6.3.0, set the <code>TEST_DEBUG</code> environment variable:
 
 ```sh
 $ TEST_DEBUG=true make test
 ```
 
+<details>
+  <summary>More options</summary>
+  Or you can use Yarn
```
Comment (JLHwung) [inline CONTRIBUTING.md:171]: ditto.

File: CONTRIBUTING.md [hunk 11]
```diff
@@ -82,64 +82,100 @@ $ make build-dist
 
 ### Running linting/tests
 
-You can run lint via:
+#### You can run lint via:
```
Comment (JLHwung) [inline CONTRIBUTING.md:85]: Don't know if we want to align the language in h4 subtitles. e.g. We can simplify put "Lint" here?

**Review by JLHwung — APPROVED** [review]
LGTM with some nits.

File: CONTRIBUTING.md (hunk 11 above)
Comment (tony-go) [reply CONTRIBUTING.md:85]: My idea was to add a bit of structure to all these lines. But we can rethink it. :)

File: CONTRIBUTING.md [hunk 12]
```diff
@@ -82,64 +82,100 @@ $ make build-dist
...
+  Substitute spaces for hyphens and forward slashes when targeting specific test names:
 
-```sh
-$ TEST_GREP="arrow functions destructuring parameters" make test
-```
+  ```sh
+  $ TEST_GREP="arrow functions destructuring parameters" make test
+  ```
+
+  Or you can use Yarn:
```
[the remark this answers is not in the document]
Comment (fedeci) [reply CONTRIBUTING.md]: The runner is actually `yarn` and does not need for `jest` to be installed globally. I am not sure if we should use "Jest" or not.

File: CONTRIBUTING.md [hunk 13]
```diff
@@ -146,12 +146,17 @@ $ TEST_GREP=transformation make test
...
 
-  Or you can use Yarn:
+  You can use:
+  ```sh
+  $ TEST_GREP="arrow functions destructuring parameters" make test
+  ```
 
+  Or you can directly use Yarn:
   ```sh
   $ yarn jest -t transformation
```
Comment (JLHwung) [inline CONTRIBUTING.md:161]: This command should be equivalent to the last one:

```
yarn jest -t "arrow functions destructuring parameters"
```

File: CONTRIBUTING.md [hunk 14]
```diff
@@ -146,12 +146,17 @@ $ TEST_GREP=transformation make test
   <summary>More options</summary>
```
Comment (JLHwung) [inline CONTRIBUTING.md:146]: Actually I am fine with removing the `details` tag, to make it more visible. Many, if not most, Babel fixture folders have `-` in their name.

Comment (tony-go) [reply CONTRIBUTING.md:146]: Yeah you right ^^

**Pull request merged**